src/meta_memcache/cache_pool.py

```diff
@@ -0,0 +1,70 @@
+from typing import Callable, Iterable, Optional, Tuple
+
+from meta_memcache.base.base_cache_pool import BaseCachePool
+from meta_memcache.commands.high_level_commands import HighLevelCommandsMixin
+from meta_memcache.commands.meta_commands import MetaCommandsMixin
+from meta_memcache.configuration import (
+    ServerAddress,
+    build_server_pool,
+    default_key_encoder,
+)
+from meta_memcache.connection.pool import ConnectionPool
+from meta_memcache.connection.providers import HashRingConnectionPoolProvider
+from meta_memcache.executors.default import DefaultExecutor
+from meta_memcache.routers.default import DefaultRouter
+from meta_memcache.routers.gutter import GutterRouter
+from meta_memcache.interfaces.cache_api import CacheApiProtocol
+from meta_memcache.protocol import Key
+from meta_memcache.serializer import BaseSerializer, MixedSerializer
+
+
+class CachePool(HighLevelCommandsMixin, MetaCommandsMixin, BaseCachePool):
+    @staticmethod
+    def cache_pool_from_servers(
```

Comment:
I think there's still some value in having `sharded` or `hash` in the method name - some libraries don't default to a sharded pool (I remember python-binary-memcached defaults to replicating keys to all servers provided and not hashing).